aemedge/blocks/tabs/tabs.css

```diff
@@ -271,3 +272,96 @@
   }
 
 }
+
+/* "big" variant */
+.tabs.big .tabs-list button {
+  font-size: 1.25rem; 
+  padding: 1rem 1.5rem; 
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+.tabs.big .tabs-list button p {
+  font-size: 1.25rem;
+  font-weight: 600;
+}
+
+.tabs.big .accordion-wrapper {
+  max-width: 1200px;
+  margin: 2rem auto;
+  padding: 0 2rem;
+}
+
+.tabs.big .accordion .details .summary {
+  padding: 1.25rem 2rem 1.25rem 0;
+  border-bottom: 1px solid rgb(23 23 37 / 10%);
+  font-size: 1.5rem; 
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.tabs.big .accordion .details .summary::before {
+  order: 2; /* Move arrow to end */
+  margin: 4px 0 0 2rem; /* Adjust margins for right alignment */
+  position: absolute;
+  right: 0;
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+.tabs.big .accordion .details .summary p {
+  order: 1;
+  flex: 1;
+  color: #454550;
+  font-weight: bold;
+  text-transform: uppercase;
+  font-size: 20px;
+}
+
+.tabs.big .accordion .details .accordion-item-body {
+  font-size: 1.125rem; 
+  padding-left: 4rem;
+  line-height: 1.75;
+}
+
+/* Media query adjustments */
+@media (width >= 768px) {
+  .tabs.big .accordion .details .accordion-item-body {
+    padding-left: .5rem;
+    padding-right: .5rem;
+  }
+
+  .tabs.big .accordion-wrapper {
+    padding: 0 3rem;
+  }
+  
+  .tabs.big .tabs-list {
+    margin-bottom: 2rem;
+  }
+}
+
+/* Additional spacing for larger screens */
+@media (width >= 992px) {
+  .tabs.big .accordion-wrapper {
+    padding: 0 4rem;
+  }
+}
+
+.tabs.big .accordion .details .accordion-item-body span.icon img,
+.tabs.big .accordion .details .accordion-item-body span.icon {
+  width: 128px;
+  height: 128px;
+  object-fit: contain;
+  display: block;
+  margin: 1rem 0;
+}
+
+
+/* Handle picture elements with regular images */
+/* stylelint-disable-next-line no-descending-specificity */
+.tabs.big .accordion .details .accordion-item-body picture img {
+  width: 128px;
+  height: 128px;
```

Comment:
I don't know how best to fix this, but because there is a JS calculated max-height on the .accordion-item-body, having the height: 128px on the image is pushing the bottom of the text out of the visible area.
![image](https://github.com/user-attachments/assets/b8d78a65-dbe8-471a-a14c-d37ad9749dbb)

Reply:
Ok, thanks -- I think that is acceptable.